frontend/common/VSCodeApi.js

```diff
@@ -2,9 +2,29 @@
 
 // @ts-ignore
 export const available = window.acquireVsCodeApi != null
-export const api = available
-    ? // @ts-ignore
-      window.acquireVsCodeApi()
-    : {
-          postMessage: console.error,
-      }
+// @ts-ignore
+const vscode = window.acquireVsCodeApi?.() ?? {}
+const set_local_cell = (cell_id, new_val) => {
```

Comment:
Rename this and the next function to something like `store_cell_input_in_vscode_state` and `load_cell_inputs_from_vscode_state`?